InAppUtils/InAppUtils.m

```diff
@@ -265,6 +267,66 @@ - (void)dealloc
     [[SKPaymentQueue defaultQueue] removeTransactionObserver:self];
 }
 
+#pragma mark Static
+
++ (NSDictionary *)parseIntroductoryPrice: (SKProduct *)product {
+    if(@available(iOS 11.2, *)) {
+        if (product != nil && product.introductoryPrice != nil) {
+            // paymentMode: Returning as string for ease of use and code resilience
+            NSString *paymentMode;
+            switch (product.introductoryPrice.paymentMode) {
+                case SKProductDiscountPaymentModeFreeTrial:
+                    paymentMode = @"freeTrial";
+                    break;
+                case SKProductDiscountPaymentModePayAsYouGo:
+                    paymentMode = @"payAsYouGo";
+                    break;
+                case SKProductDiscountPaymentModePayUpFront:
+                    paymentMode = @"payUpFront";
+                    break;
+                default:
+                    paymentMode = @"unavailable";
+                    break;
+            }
+            
+            // subscriptionPeriod: Returning as Dictionary { unit: NSString, numberOfUnits: NSNumber }
+            NSString *subscriptionPeriodUnit;
+            switch (product.introductoryPrice.subscriptionPeriod.unit) {
+                case SKProductPeriodUnitDay:
+                    subscriptionPeriodUnit = @"day";
+                    break;
+                case SKProductPeriodUnitWeek:
+                    subscriptionPeriodUnit = @"week";
+                    break;
+                case SKProductPeriodUnitMonth:
+                    subscriptionPeriodUnit = @"month";
+                    break;
+                case SKProductPeriodUnitYear:
+                    subscriptionPeriodUnit = @"year";
+                    break;
+                default:
+                    subscriptionPeriodUnit = @"unavailable";
+                    break;
+            }
+            
+            NSDictionary *subscriptionPeriod = @{
+                                                 @"unit": subscriptionPeriodUnit,
+                                                 @"numberOfUnits": [[NSNumber alloc] initWithLong:product.introductoryPrice.subscriptionPeriod.numberOfUnits],
+                                                 };
+            
+            NSDictionary *introductoryPrice = @{
+                                                @"price": product.introductoryPrice.price,
```

Comment:
It would be nice to get the localized price string here. Similar to the product implementation.

Reply:
I couldn't add my commits in this existing PR so I made new one with the changes you requested (#206)